dashboard/api.py

```diff
@@ -498,8 +502,10 @@ def get_student_current_grades(user, edx_client):
     refresh_delta = now - datetime.timedelta(hours=REFRESH_GRADES_CACHE_HOURS)
 
     with transaction.atomic():
+        enrolled_curses = models.CachedEnrollment.objects.filter(user=user).exclude(data=None).values_list(
```

Comment:
It seems like this would create a dependency where `get_student_enrollments` must get run first or else some grades may not get fetched. Can you add a comment saying this in `dashboard/tasks.py` and `dashboard/views.py` where we call the function?